pkg/logs/config/source.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package config
+
+import "sync"
+
+// LogSource holds a reference to and integration name and a log configuration, and allows to track errors and
+// successful operations on it. Both name and configuration are static for now and determined at creation time.
+// Changing the status is designed to be thread safe.
+type LogSource struct {
+	Name   string
+	Config *LogsConfig
+	Status *LogStatus
+	inputs map[string]bool
+	lock   *sync.Mutex
+}
+
+// NewLogSource creates a new log source.
+func NewLogSource(name string, config *LogsConfig) *LogSource {
+	return &LogSource{
+		Name:   name,
+		Config: config,
+		Status: NewLogStatus(),
+		inputs: make(map[string]bool),
+		lock:   &sync.Mutex{},
+	}
+}
+
+// AddInput registers an input as being handled by this source.
+func (s *LogSource) AddInput(input string) {
```

Comment:
I think there is a lake of information here, what's an input ? I'd rather use `Instance`

Reply:
`Input` felt rather generic to me as a meta-name to describe things we tail (files, containers). If you are strongly against it I'll consider changing it, but I'm not that convinced by `Instance`.

Reply:
I think it's fine